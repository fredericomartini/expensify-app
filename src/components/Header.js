import React from 'react';
import Menu from './Menu';

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <Menu />
  </header>
);

export default Header;
