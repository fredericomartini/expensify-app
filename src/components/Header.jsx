import { Component } from 'react';

class Header extends Component {
    render() {
        return (
            <p>I`m a Header!!</p>
        );
    }
}

export default Header;