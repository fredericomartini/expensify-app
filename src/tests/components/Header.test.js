import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

test('should RENDER HEADER CORRECTLY', () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
