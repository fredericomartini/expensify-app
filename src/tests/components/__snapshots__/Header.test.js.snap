// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should RENDER HEADER CORRECTLY 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <Menu />
</header>
`;
